documentation/CREDITS.rst

```diff
@@ -13,7 +13,7 @@ Contributors
     volume techniques, and level set treatments.
```

Comment:
Should we get rid of this as well and just point people at the Github profiles? This is so out of date. Another issue?

Reply:
Beyond the scope of this PR. Ticket filed.